clang/test/CodeGen/call-graph-section.c

```diff
@@ -0,0 +1,83 @@
+// Tests that we assign appropriate identifiers to indirect calls and targets.
+
```

Comment:
```suggestion
REQUIRES: x86-registered-target
```
I think you need a requires line here, if you're gong to set the triple. Or this could be under `CodeGen/X86`. Maybe its OK because we stop at IR generation? I don't recall precisely, but there are at least some files that only emit LLVM IR and use REQUIRES in this folder, so it seems safe to do it that way.